.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Docker
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+jobs:
+  push:
```

Comment:
Hey

I like the dockerfile but not sure what this is for, is it necessary?

Reply:
The purpose of that workflow is to automatically publish a new dockerized version of Bolang on every update on master. I'm using the same workflow to publish a new version of dockerized my services in some projects. Screen rel -> https://imgur.com/a/XBoQN1X.